matscipy/calculators/manybody/calculator.py

```diff
@@ -809,3 +723,66 @@ def get_non_affine_forces(self, atoms):
             + mabincount(j_p[ik_t], naF53_tcab, minlength=nb_atoms)
 
         return naforces_icab / 2
+
+
+class NiceManybody(Manybody):
```

Comment:
This will become that standard interface, correct?

Reply:
Ultimately yes. I'll make another PR for a revised interface, where we calculate things more efficiently.